website/docs/reference/resource-properties/data-tests.md

```diff
@@ -29,7 +29,8 @@ models:
   - name: <model_name>
     data_tests:
       - [<test_name>](#custom-data-test-name):
-          <argument_name>: <argument_value>
+          arguments: # available in v1.10 and higher
```

Comment:
Since there could feasibly be users reading this reference and configuring on older versions, it might be helpful to make this text more descriptive so they don't misconfigure. Maybe something like this (but run it by an SME to be sure):
```suggestion
          arguments: # available in v1.10 and higher. Older versions can set the <argument_name> as the top-level property.
```